api/organisations/tests/test_chargebee.py

```diff
@@ -7,21 +7,24 @@
 from organisations.chargebee import (
     get_customer_id_from_subscription_id,
     get_hosted_page_url_for_subscription_upgrade,
+    get_max_api_calls_for_plan,
     get_max_seats_for_plan,
+    get_plan_meta_data,
     get_portal_url,
     get_subscription_data_from_hosted_page,
 )
 
 
 class MockChargeBeePlanResponse:
-    def __init__(self, max_seats=0):
-        self.max_seats = max_seats
-        self.plan = MockChargeBeePlan(max_seats)
+    def __init__(self, max_seats=0, max_api_calls=50000):
+        self.max_seats = (max_seats,)
```

Comment:
Why is this now a tuple?

Reply:
Looking at the rest of the code - I'm not sure these class attributes are even used are they?